WooCommerce/Classes/Authentication/Epilogue/StorePickerError.swift

```diff
@@ -1,21 +1,62 @@
 import SwiftUI
+import SafariServices
 
 /// Hosting controller wrapper for `StorePickerError`
 ///
 final class StorePickerErrorHostingController: UIHostingController<StorePickerError> {
+
+    /// Creates an `StorePickerErrorHostingController` with preconfigured button actions.
+    ///
+    static func createWithActions(presenting: UIViewController) -> StorePickerErrorHostingController {
```

Comment:
Nit: `presenting` sounds a bit ambiguous to me. Do you think `presentingViewController` may be clearer?

Reply:
Not sure, I took the name from the [UIViewControllerTransisioningDelegate protocol](https://developer.apple.com/documentation/uikit/uiviewcontrollertransitioningdelegate/1622037-animationcontroller) which call their parameters `presenting` and `presented`. So I guess it's fairly standard!